zulipterminal/ui_tools/buttons.py

```diff
@@ -174,6 +174,9 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
                 )
         elif is_command_key('TOGGLE_MUTE_STREAM', key):
             self.controller.stream_muting_confirmation_popup(self)
+        elif is_command_key('MSG_INFO', key):
```

Comment:
You've used `MSG_INFO` here and elsewhere, which works fine since it's the same key.

However, it would be useful to have this separately documented in the help menu and in the README; adding it manually to the latter and a new entry to the key metadata would be one way to resolve this.